environments.json

```diff
@@ -1496,23 +1496,32 @@
     "family": "V8",
     "short": "CH&nbsp;61,<br>OP&nbsp;48",
     "release": "2017-09-05",
+    "retire": "2017-10-17",
     "note_id": "experimental-flag",
     "obsolete": false
```

Comment:
This should be `true` now.

Reply:
@afmenez See https://github.com/kangax/compat-table/pull/1201#discussion_r145312366

Reply:
I disagree and commented there.  Also, the retire date should be when all the new versions are released, Android and iOS are still missing.